kube/deployments/smoke-test.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       containers:
       - name: web-console-smoke-test
         image: benjaminapetersen/web-console-smoke-test:latest
-        imagePullPolicy: IfNotPresent
+        imagePullPolicy: Always
```

Comment:
We probably want `IfNotPresent` everywhere (except maybe to test)

Reply:
+1 yeah that makes sense.  I changed it as I've been testing, but let me put that back.
Should be configurable in the `openshift` template?

Reply:
I'd rather limit options for things don't really need to be configurable. You can also edit the deployment if you really need to. You could leave `imagePullPolicy` off entirely if you want it to pull `:latest` always, but not other tags.

Reply:
For development on a local cluster, you can build the image locally and delete the pod without needing to push the image (as long as you leave this `IfNotPresent`).

I'd prefer this way of testing instead of using `docker run`

Reply:
Updated to `IfNotPresent`.

I've tested quite a bit either way, mostly `oc process | oc create` these days.